### PR TITLE
improve android wizard

### PR DESCRIPTION
**Pull request by bruno-garcia**
I rather hard code the version now than make people go to GitHub.

We **need** to add craft on this repo.

### CODE REVIEW

**Review by dcramer — APPROVED** [review]
You could add `<PageGrid nextSteps header="Next Steps" />` as an alternative on this page.

**Comment by bruno-garcia** [comment]
Will do. Thanks.

File: src/wizard/android/index.md [hunk 1]
```diff
@@ -52,11 +51,13 @@ Great! Now that you’ve completed setting up the SDK, maybe you want to quickly
...
+        Sentry.captureMessage("testing SDK setup");
     }
 }
 ```
+
+### Next steps
+
+Using ProGuard or R8 to obfuscate your app? Check out [our docs on how to set it up](https://docs.sentry.io/platforms/android/proguard/).
+
+[The documentation](https://docs.sentry.io/platforms/android/configuration/) has more information about the many configurations and API available in Sentry's SDK.
```
Comment (bruno-garcia) [inline src/wizard/android/index.md:63]:
```suggestion
<PageGrid nextSteps header="Next Steps" />
```

**Comment by dcramer** [comment]
jk this is wizard it wont work sorryyyyy

**Comment by bruno-garcia** [comment]
hehe np, just realized after I pushed it

File: src/wizard/android/index.md [hunk 2]
```diff
@@ -52,11 +51,9 @@ Great! Now that you’ve completed setting up the SDK, maybe you want to quickly
...
-            throw new Exception("This is a test.");
-        } catch (Exception e) {
-            Sentry.captureException(e);
-        }
+        Sentry.captureMessage("testing SDK setup");
     }
 }
 ```
+
+<PageGrid nextSteps header="Next Steps" />
```
Comment (bruno-garcia) [inline src/wizard/android/index.md:59]:
```suggestion
### Next steps

Using ProGuard or R8 to obfuscate your app? Check out [our docs on how to set it up](https://docs.sentry.io/platforms/android/proguard/).

[The documentation](https://docs.sentry.io/platforms/android/configuration/) has more information about the many configurations and API available in Sentry's SDK.
```

**Pull request merged**